### PR TITLE
Upgrade to 0.4.5

### PR DESCRIPTION
**Pull request by iameskild**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by iameskild** [comment]
@conda-forge-admin, please rerender

**Comment by iameskild** [comment]
@conda-forge-admin, please rerender

**Review by viniciusdc — APPROVED** [review]
Thanks for adding the extra dependencies @iameskild

**Comment by viniciusdc** [comment]
@conda-forge-admin, please rerender

**Comment by viniciusdc** [comment]
@conda-forge-admin, please restart ci

File: recipe/meta.yaml [hunk 1]
```diff
@@ -32,12 +32,15 @@ requirements:
     - pydantic
     - pynacl
     - python-keycloak
+    - requests-toolbet ==0.9.1
```
Comment (viniciusdc) [inline recipe/meta.yaml:35]:
```suggestion
    - requests-toolbet <0.10.0
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -32,12 +32,15 @@ requirements:
     - pydantic
     - pynacl
     - python-keycloak
+    - requests-toolbet <0.10.0
```
Comment (viniciusdc) [inline recipe/meta.yaml:35]:
```suggestion
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -32,12 +32,15 @@ requirements:
     - pydantic
     - pynacl
     - python-keycloak
```
Comment (viniciusdc) [inline recipe/meta.yaml:34]:
```suggestion
    - python-keycloak >=2.5.0,<2.6.0
```

Comment (viniciusdc) [reply recipe/meta.yaml]: This needs to be added to force the pin you wanted, unfortunately the smithy doesn't seem to find the correct version of `requests-toolbet` from within conda-forge

File: recipe/meta.yaml [hunk 4]
```diff
@@ -38,6 +38,8 @@ requirements:
     - azure-mgmt-containerservice ==16.2.0
     - importlib_metadata
     - rich
+    - typer
```
Comment (viniciusdc) [inline recipe/meta.yaml:41]:
```suggestion
    - requests-toolbelt<0.10.0,>=0.9.1
    - typer
```

File: recipe/meta.yaml (hunk 3 above)
Comment (viniciusdc) [reply recipe/meta.yaml]: disregard this comment, as it seems that pinning Keycloak alone didn't fix the issue

File: recipe/meta.yaml [hunk 5]
```diff
@@ -38,6 +38,9 @@ requirements:
     - azure-mgmt-containerservice ==16.2.0
     - importlib_metadata
     - rich
+    - requests-toolbelt<0.10.0,>=0.9.1
```
Comment (viniciusdc) [inline recipe/meta.yaml:41]:
```suggestion
    - requests-toolbelt <0.10.0,>=0.9.1
```

**Pull request merged**